src/angular/dialog/dialog.scss

```diff
@@ -0,0 +1,96 @@
+@use 'external/npm/node_modules/@angular/cdk' as cdk;
+@import '../styles/common';
+
+$sbb-dialog-padding: pxToRem(16);
+$sbb-dialog-button-margin: pxToRem(8);
+
+.sbb-overlay-background {
+  background-color: rgba(255, 255, 255, 0.7);
+  align-items: center;
+}
+
+.sbb-dialog-container {
+  display: block;
+  border: 1px solid $sbbColorIron;
```

Comment:
I'm uncertain if it would be nice to provide standard borders as well? I personally would do it, but I know, it's a lean only component